src/test/scala/PropertyCheckerTest.scala

```diff
@@ -1040,42 +1040,44 @@ class PropertyCheckerTest extends FunSuite {
   test(
     "should insert pattern key under format if format is textual for Numeric format datatypes"
   ) {
-    val json =
-      """
-        |{
-        | "base": "decimal",
-        | "format": "0.###E0"
-        |}
-        |""".stripMargin
-
-    val jsonNode = objectMapper.readTree(json)
-    val (values, warnings, _) =
-      PropertyChecker.checkProperty("datatype", jsonNode, "", "und")
-
-    // format object should contain the key pattern
-    assert(!values.path("format").get("pattern").isMissingNode)
+    // Update: Not validating numeric data-types with formats
+//    val json =
+//      """
+//        |{
+//        | "base": "decimal",
+//        | "format": "0.###E0"
+//        |}
+//        |""".stripMargin
+//
+//    val jsonNode = objectMapper.readTree(json)
+//    val (values, warnings, _) =
+//      PropertyChecker.checkProperty("datatype", jsonNode, "", "und")
+//
+//    // format object should contain the key pattern
+//    assert(!values.path("format").get("pattern").isMissingNode)
```

Comment:
Again, if we're not testing something, let's comment the whole test out so it doesn't look like it's running when there's nothing to do.